server/protocols/webadmin/webadmin-cassandra/src/test/java/org/apache/james/webadmin/routes/CassandraMigrationRoutesTest.java

```diff
@@ -267,7 +266,7 @@ public void postShouldReturnTaskIdAndLocation() throws Exception {
 
     @Test
     public void createdTaskShouldHaveDetails() throws Exception {
-        when(schemaVersionDAO.getCurrentSchemaVersion()).thenReturn(CompletableFuture.completedFuture(Optional.of(OLDER_VERSION)));
+        when(schemaVersionDAO.getCurrentSchemaVersion()).thenReturn(Mono.just(Optional.of(OLDER_VERSION)));
```

Comment:
`Mono::equals` is not documented (and not part of the Mono API) and thus looks implementation specific. Under the hood, these is a hidden call to .block().

I would prefer:

```
assertThat(x.block()).isEqualTo(y);
```

Rather than

```
assertThat(x).isEqualTo(Mono.just(y));
```

Or... Maybe another way would be to write our own asserts for Mono?

```
assertThat(x).completesWithValue(y);
```

Reply:
It's not an equal, it's a mock, I do not know what do to then :/